crates/scalajs/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "wit-bindgen-scalajs"
+authors = ["Daniel Vigovszky <me@vigoo.dev>"]
+version = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+homepage = 'https://github.com/bytecodealliance/wit-bindgen'
+description = """
+Scala.JS bindings generator for WIT and the component model, typically used
```

```suggestion
Scala.js bindings generator for WIT and the component model, typically used
```
FYI, the recommended spellings are:

* Free text: Scala.js
* PascalCase: ScalaJS
* camelCase: scalaJS
* URL: scala-js